cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -3242,6 +3242,9 @@
                     "stack_name": {
                       "Ref": "AWS::StackName"
                     },
+                    "enable_efa": {
```

Comment:
enable_efa is not set in the /tmp/dna.json for the master server which might make this conditional check in the cookbook to fail: `if node['cfncluster']['enable_efa'] == 'compute' && node['cfncluster']['cfn_node_type'] == 'ComputeFleet'`.

We need to either check for ComputeFleet before enable_efa or set the value for the master server too. I would probably go for the second option and keep the dna.json aligned for master and compute nodes.

Reply:
Added